frontend/src/modules/Shared/Shares/ShareEditForm.js

```diff
@@ -12,7 +12,12 @@ import {
   TextField,
   Typography
 } from '@mui/material';
-import { Defaults, Pager, ShareStatus } from '../../../design';
+import {
+  Defaults,
+  Pager,
+  ShareHealthStatus,
+  ShareStatus
+} from '../../../design';
```

Comment:
Although it was not introduced in this PR, could you change the relative imports from design to absolute design? 
import 
```
import {
  Defaults,
  Pager,
  ShareHealthStatus,
  ShareStatus
} from 'design';
```